internal/cache/cachedb.go

```diff
@@ -25,6 +25,7 @@ const (
 	defaultCachePath = "/var/lib/aad/cache"
 	passwdDB         = "passwd.db" // root:root 644
 	shadowDB         = "shadow.db" // root:shadow 640
+	db_con_args      = "?_journal_mode=wal"
```

Comment:
Go does not allow snake casing in variable names, would you mind refactoring this?
```suggestion
	dbConnArgs      = "?_journal_mode=wal"
```

Reply:
Hello, thanks for the feedback. fixed in latest commit. :)